server/neptune/workflows/internal/activities/audit.go

```diff
@@ -0,0 +1,57 @@
+package activities
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/deploy/request"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/job"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/root"
+)
+
+type writer interface {
+	Write([]byte) error
+}
+
+type auditActivities struct {
+	SnsWriter writer
+}
+
+type AuditJobRequest struct {
+	DeploymentInfo root.DeploymentInfo
+	State          job.State
+}
+
+func (a *auditActivities) AuditJob(ctx context.Context, req AuditJobRequest) error {
+	isForceApply := req.DeploymentInfo.Root.Trigger == root.Trigger(request.ManualTrigger)
+	atlantisJobEvent := &job.Event{
+		Version:        1,
+		ID:             req.DeploymentInfo.ID,
```

Comment:
Just wanted to confirm that it's intentional to use the DeploymentInfo id instead of the Job id that we originally used/no issues with changing it.

Reply:
I looked up the references for the Job Event's ID and `atlantisJobID` is not being used any where except when updating the atlantis job in [deployapi](https://github.com/lyft/deployapi/blob/a5e4f4d4cfcb9fba942b67f564a73303525fec48/sqsqueue/model.go#L58). The compliance violation store does not use this ID either. Afaik, I don't think changing this to deploymentInfo id should be an issue. I'll start a thread in our channel to get some more input on this.